v3/src/data-interactive/handlers/global-handler.ts

```diff
@@ -0,0 +1,61 @@
+import { kDefaultSliderName, kDefaultSliderValue } from "../../components/slider/slider-utils"
+import { appState } from "../../models/app-state"
+import { getGlobalValueManager, getSharedModelManager } from "../../models/tiles/tile-environment"
+import { t } from "../../utilities/translation/translate"
+import { registerDIHandler } from "../data-interactive-handler"
+import { DIGlobal, DIHandler, DIResources, DIValues } from "../data-interactive-types"
+import { valuesFromGlobal } from "../di-conversion-utils"
+
+const illegalValueResult = { success: false, values: { error: t("V3.DI.Error.globalIllegalValue") } } as const
+
+export const diGlobalHandler: DIHandler = {
+  create(_resources: DIResources, values?: DIValues) {
+    const { document } = appState
+    const globalManager = getGlobalValueManager(getSharedModelManager(document))
+    const { name, value } = values as DIGlobal
+
+    const _value = Number(value ?? kDefaultSliderValue)
+    if (isNaN(_value)) return illegalValueResult
+
+    const globalSnapshot = {
+      name: (name ?? globalManager?.uniqueName() ?? kDefaultSliderName).toString(),
```

Comment:
```suggestion
      name: (name || globalManager?.uniqueName() || kDefaultSliderName).toString(),
```
We shouldn't allow empty strings, either.